.refactored/sciserver-rstudio/2.0/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+nginx
+
+cd /home/idies/workspace
+
+exec sudo -u idies env PATH=$PATH /usr/lib/rstudio-server/bin/rserver
```

Comment:
why not leave the `sudo` out and just set `USER idies` in the dockerfile?

Reply:
Might then need to allow idies to run nginx, which I'm not sure we should in principle allow?